src/frontend/src/lib/components/core/Loader.svelte

```diff
@@ -69,10 +71,14 @@
 	let progressModal = false;
 
 	const debounceLoadBtcAddressTestnet = debounce(loadBtcAddressTestnet);
+	const debounceLoadBtcAddressRegtest = debounce(loadBtcAddressRegtest);
 
 	$: {
 		if ($testnets && isNullish($btcAddressTestnet)) {
 			debounceLoadBtcAddressTestnet();
+			if (LOCAL) {
```

Comment:
Shoul we have a single debounced function and within that function add the logic of which addresses to load?

Or it's better two debounce multiple functions for this case?

Reply:
No strong opinion, I didn't think this would matter. Because it was called loadAddressTestnet I thought of using two. If it would have been called loadAddressTest I would have put it inside.

Reply:
The questions is probably about what happens if one or the other does not load. Like is it fine if one fail and the other not?

If the former, then it is indeed opiniated but, if the latter, we need to express it.

In both case it's probably more readable to have `await Promise.all` or `allSettled`?

Kind of a half opiniated / half meaningful feedback 😄

Reply:
It's fine if one fails and not the other.

We were not doing anything here with the outcome before, nor we do anything now.

I can change the logic though. How do you want me to change it?

Reply:
> It's fine if one fails and not the other.

Great. Then it's maybe a bit an opiniated answer so I gonna approve the PR and let you decide if you want to adapt or not but, I would maybe do following which kind of reflect the other function of the component called `loadData`:

```
const loadBtcAddress = async () = {
   await Promise.allSettled([
			loadBtcAddressTestnet(),
			...(LOCAL ? [loadBtcAddressRegtest()] : [])
		]);
}
const loadBtcAddress = debounce(loadBtcAddress);

$: {
		if ($testnets && isNullish($btcAddressTestnet)) {
		     loadBtcAddress();
		}
	}
```